packages/core/src/components/templates/UnsupportedField.tsx

```diff
@@ -16,17 +18,25 @@ function UnsupportedField<
   S extends StrictRJSFSchema = RJSFSchema,
   F extends FormContextType = any
 >(props: UnsupportedFieldProps<T, S, F>) {
-  const { schema, idSchema, reason } = props;
+  const { schema, idSchema, reason, registry } = props;
+  const { translateString } = registry;
+  let translateEnum: TranslatableString = TranslatableString.UnsupportedField;
+  const translateParams: string[] = [];
+  if (idSchema && idSchema.$id) {
+    translateEnum = TranslatableString.UnsupportedFieldWithId;
+    translateParams.push(idSchema.$id);
+  }
+  if (reason) {
+    translateEnum =
+      translateEnum === TranslatableString.UnsupportedField
+        ? TranslatableString.UnsupportedFieldWithReason
+        : TranslatableString.UnsupportedFieldWithIdAndReason;
+    translateParams.push(reason);
+  }
   return (
     <div className="unsupported-field">
       <p>
-        Unsupported field schema
-        {idSchema && idSchema.$id && (
-          <span>
-            {" for"} field <code>{idSchema.$id}</code>
-          </span>
-        )}
-        {reason && <em>: {reason}</em>}.
+        <Markdown>{translateString(translateEnum, translateParams)}</Markdown>
```

Comment:
Hopefully this doesn't introduce any XSS-like risks?

Reply:
Should we enable markdown support for all localized strings?

Reply:
This and `FileWidget` the only place where we were constructing code with embedded HTML in them. I believe that `Markdown` prevents XSS via this:
```
Arbitrary HTML is supported and parsed into the appropriate JSX representation without dangerouslySetInnerHTML
```

Reply:
> Should we enable markdown support for all localized strings?

I hesitate because some of them need to be strings. I can add comments to the two enums indicating they support Markdown and html